src/app/examples/segmented-control-example/segmented-control-example.component.html

```diff
@@ -0,0 +1,11 @@
+<kirby-segmented-control [segmentItems]="segmentItems" (segmentClicked)="onSegmentClick($event)"></kirby-segmented-control>
```

Comment:
Use same syntax as `click`, `tap` etc:
```suggestion
<kirby-segmented-control [segmentItems]="segmentItems" (segmentClick)="onSegmentClick($event)"></kirby-segmented-control>
```